tests/pamiq_core/test_state_persistence.py

```diff
@@ -79,3 +83,42 @@ def test_load_state(self, tmp_path, mocker):
 
         mock_state_1.load_state.assert_called_once_with(tmp_path / "mock_state_1")
         mock_state_2.load_state.assert_called_once_with(tmp_path / "mock_state_2")
+
+
+class TestPickleFunctions:
+    @pytest.fixture
+    def temp_file(self, tmp_path):
+        """Fixture to provide a temporary file path."""
+        return tmp_path / "test.pkl"
+
+    @pytest.fixture
+    def sample_data(self):
+        """Fixture to provide sample data for testing."""
+        return {"name": "test", "values": [1, 2, 3], "nested": {"a": 1, "b": 2}}
+
+    def test_save_and_load_pickle(self, temp_file, sample_data):
+        """Test saving and loading an object with pickle."""
+        save_pickle(sample_data, temp_file)
+
+        # Verify file exists
+        assert temp_file.is_file()
+
+        # Load and verify data
+        loaded_data = load_pickle(temp_file)
+        assert loaded_data == sample_data
+
+    def test_save_pickle_with_string_path(self, temp_file, sample_data):
+        """Test saving pickle using a string path."""
+        save_pickle(sample_data, str(temp_file))
+
+        # Verify file exists
+        assert temp_file.is_file()
+
+        # Load and verify data
+        loaded_data = load_pickle(temp_file)
```

Comment:
こちらも引数は str のほうが良かったりしないでしょうか？

（超厳密には、save が Path or str、load が Path or str の 2 x 2 = 4 通りのテストケースがあったほうがいいかなとも思いますが、
まぁ流石にやりすぎかなとは思うので、
この2ケースでいいとおもいつつ、load でも str を試したほうが良いかと思います）

```suggestion
        loaded_data = load_pickle(str(temp_file))
```

Reply:
ありがとうございます！